docs/assets/css/main.css

```diff
@@ -0,0 +1,2321 @@
+/*! normalize.css v1.1.3 | MIT License | git.io/normalize */
```

Comment:
This file is copied from `normalize.css`. It should live in that repo, not copied to this repo.